core/src/klio_core/testing.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Spotify AB
```

Comment:
Are you intending that users can use objects from this module? If not, can we make it `_testing.py`?

If you are, I would want it `klio` since `klio-core` isn't intended for users (although I understand that screws up dependencies for the klio-cli).

Reply:
Oh yeah I don't think users need to use this, will change to `_testing.py`